proposals/0038-16bit-isspecialfloat.md

```diff
@@ -14,31 +14,42 @@
 
 ## Introduction
 
-The IsSpecialFloat DXIL Op is used to implement operations, 'isinf', 'isnan', 'isfinite', and 'isnormal'.
-Due to a bug (#7496), the IsSpecialFloat DXIL operations were never generated for 16-bit types.
+The IsSpecialFloat DXIL Op is used to implement operations, 'isinf', 'isnan',
+'isfinite', and 'isnormal'. Due to a bug (#7496), the IsSpecialFloat DXIL
+operations were never generated for 16-bit types.
 
 ## Motivation
 
-The 'isinf', 'isnan', and 'isfinite' functions (there is no 'isnormal' function), support fp16 but instead of generating a 16 bit
-IsSpecialFloat DXIL Op, DXC extends to the 32 bit float op. See (https://github.com/microsoft/DirectXShaderCompiler/issues/7496).
+The 'isinf', 'isnan', and 'isfinite' functions (there is no 'isnormal' function)
+, support fp16 but instead of generating a 16 bit IsSpecialFloat DXIL Op,
```

Comment:
```suggestion
The 'isinf', 'isnan', and 'isfinite' functions (there is no 'isnormal' function),
support fp16 but instead of generating a 16 bit IsSpecialFloat DXIL Op,
```